Helper/Data.php

```diff
@@ -101,25 +104,31 @@ protected function _getConfigValue($path, $contextScope, $contextScopeId) {
      * @param string|int $contextScopeId
      * @return array
      */
-    public function getConfigDisplayValue($path, $contextScope, $contextScopeId) {
+    public function getConfigDisplayValue($path, $contextScope, $contextScopeId)
+    {
         $value = $this->_getConfigValue($path, $contextScope, $contextScopeId);
 
-        $labels = [$value]; //default labels to raw value
+        if (is_array($value)) {
+            $firstKey = array_keys($value)[0];
```

Comment:
You might also use the [key](http://php.net/key) function, which seems simpler and easier to read.
```suggestion
            $firstKey = key($value);
```